packages/babel-cli/src/babel/dir.js

```diff
@@ -104,7 +154,10 @@ export default function(commander, filenames, opts) {
         watcher.on(type, function(filename) {
           const relative = path.relative(dirname, filename) || filename;
           try {
-            handleFile(filename, relative);
+            handleFile(filename, relative, function(err, res) {
+              if (err) throw err;
+              util.log(res);
```

Comment:
Looks like this may be a mistake? There doesn't appear to be a `res` value passed to this callback unless I'm misreading, and there wasn't a lot here before.

Reply:
You're right. I've removed it.